github.user.styl

```diff
@@ -950,38 +952,64 @@ for c, i in C {
 
         &[type='radio'],
         &[type='checkbox'] {
-            &, &:focus, &:hover:active, &:disabled, &:hover:active:disabled {
+            &, &:focus, &:hover:active {
                 -webkit-appearance: none
                 -moz-appearance: none
                 appearance: none
                 cursor: pointer
-                height: 1rem
-                width: 1rem
+                height: b_hgwd
+                width: b_hgwd
                 border: 1px solid bg5
                 background-size: cover
                 c: fg1 0 bg1
             }
+            &:disabled, &:hover:active:disabled {
+                c: fg1 0 bg4
+            }
         }
 
-        &[type='radio']:checked {
-            z = '<circle cx="256" cy="256" r="128" fill="white"/>'
-            background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-            border-radius: 100% i
-            &:checked { c: 0 ac4 ac4 }
+        &[type='radio'] {
+            /// fix border radius at https://github.com/settings/profile Emoji color
+            border-radius: 50%
+            &:checked {
+                z = '<circle cx="256" cy="256" r="128" fill="white"/>'
+                background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
+                c: 0 ac2 bg2
+            }
+            &:focus { border-radius: 50% }
+            &:hover { c: 0 ac1 bg2 }
```

Comment:
I like the idea, it's just weird. Inputs are larger than checkboxes, so the hover styles apply even when the cursor is not hovering over the checkbox. This sort of adds on top of my previous comment.

Reply:
> Also, it looks like I forgot to push my previous commits to GitHub, which now causes a conflict because of the commit [e208150](https://github.com/vednoc/dark-github/commit/e20815089e2802e3f97e1a5b0e47273280681d43) that changes border/background colors for checkboxes. 🤦🏻‍♂️

no idea, if you push whatever it was I can rebase the pr on main